src/Button/button-group.scss

```diff
@@ -36,14 +36,64 @@
     margin-left: calc(var(--pgn-size-btn-border-width) * -1);
   }
 
+  // Mixin for button states that affect border radius
+
+  @mixin button-border-radius-0($side) {
```

Comment:
Should this mixin be in the button [mixins.scss](https://github.com/openedx/paragon/blob/ags/elm-theme-standalone/src/Button/_mixins.scss)?

Reply:
Yes, I agree it should! I will move it to the button `mixins.scss` per your suggestion. Just a heads up, I'm splitting these `ButtonGroup` changes into their own distinct PR, separate from this one! I'll tag you on that PR with these changes once it's up 😄

Reply:
Split out PR for `ButtonGroup`: https://github.com/openedx/paragon/pull/3596